packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -18,15 +18,13 @@ import { BaseDeployIO } from "scripts/utils/BaseDeployIO.sol";
 // complement the OP Stack Manager. The script architecture is a bit different than a standard forge
 // deployment script.
 //
-// There are three categories of users that are expected to interact with the scripts:
-//   1. End users that want to run live contract deployments.
```

Comment:
I think we should keep the end users item, but just indicate they are expected to run the scripts via op-deployer, which also uses the go interface

Reply:
Added here, https://github.com/ethereum-optimism/optimism/pull/12015/commits/23fcc6cc37caf84b41f05ad5b5b625c6efc05e4e feel free to add more details if you have them to this. Otherwise, I can resolve and merge.